source/mainnet/docs/desktop-wallet/install-ledger-app.rst

```diff
@@ -48,378 +52,437 @@ The LEDGER device will generate the unique 24-word recovery phrase that is used
+            .. group-tab:: Nano S
 
-3. The LEDGER device says **Concordium**. Press both buttons. The LEDGER device says **Concordium is ready**. You can now use the LEDGER device with the Desktop Wallet.
+               .. code-block:: console
+                  :substitutions:
 
-.. _update-app-ubuntu:
+                  $cd ~/Downloads/concordium-governance-ledger-app-|ccd-governance-version|-nanos-|nano-s-firmware|
 
-Update the Concordium LEDGER app on Ubuntu
--------------------------------------------
+            .. group-tab:: Nano S+
 
-For the app to work properly with the current version of the Desktop Wallet, make sure that you update to the latest version of the app.
+               .. code-block:: console
+                  :substitutions:
 
-.. Note::
-    If you're using a different computer than the one you used when you installed the app, you must :ref:`install Python3, pip, and the Python tools <install-python-pip-ubuntu>` for LEDGER (ledgerblue) before updating the app.
+                  $cd ~/Downloads/concordium-governance-ledger-app-|ccd-governance-version|-nanosplus
 
-#. :ref:`Download <downloads>` the latest version of the LEDGER app if you haven't done so already.
+      #. Install the Concordium application on the LEDGER device by running the following script from the folder you extracted the files to:
 
-#. Run the ``install.sh`` file from the folder that you extracted the files to.
+         .. code-block:: console
 
-.. Note::
+            $./install.sh
+
+         .. Note::
+
+            If you get the error ``loadApp.py: error: unrecognized arguments: --apiLevel 1`` this means that you have an older version of the Python tools for LEDGER (ledgerblue). To update, run ``$pip3 install ledgerblue --upgrade``.
+
+      #. The LEDGER device says **Deny unsafe manager**. Press the right button to navigate through the public key until the LEDGER device says **Allow unsafe manager**.
+
+      #. Before you can install the new version of the LEDGER app, you have to uninstall the old one. The LEDGER device says **Uninstall Concordium**. Press the right button to navigate through the identifier until the LEDGER device says **Confirm action**. Press both buttons to confirm. The LEDGER device says **Loading, please wait** while it installs the app.
+
+      #. The LEDGER device says **Concordium**. Press both buttons. The LEDGER device says **Concordium is ready**. Press the left button to verify that you've installed the latest version of the LEDGER app.
+
+   .. group-tab:: Ubuntu
+
+      .. _update-app-ubuntu:
+
+      .. Note::
+          If you're using a different computer than the one you used when you installed the app, you must :ref:`install Python3, pip, and the Python tools <install-python-pip-ubuntu>` for LEDGER (ledgerblue) before updating the app.
+
+      #. :ref:`Download <downloads>` the latest version of the LEDGER app if you haven't done so already.
```

Comment:
could link to the section `https://docs.concordium.com/en/mainnet/docs/desktop-wallet/install-ledger-app.html#download-concordium-governance-ledger-app`

Reply:
Fixed, good catch!